SeQuant/domain/mbpt/models/cc.hpp

```diff
@@ -9,12 +9,17 @@
 namespace sequant::mbpt::sr {
 
 /// derives equations of traditional coupled-cluster method
-class cceqs {
+class cc {
```

Comment:
- classes in VG codes usually are capitalized (`Chef`), functions are lower-case (`Chef::make_cake`) ... namespaces are usually lower-case (`mpqc`, `sequent`) but sometimes upper (`TiledArray`).
- so I recommend making this `CC`